model/schema/reference.yaml

```diff
@@ -199,122 +191,125 @@ slots:
+    description: >-
+      Plain language abstract.
+      Only relevant for PubMed references.
     multivalued: false
     domain: Reference
     range: string
 
   pubmed_abstract_languages:
-    description: Languages for the abstract
+    description: >-
+      Languages for the abstract.
+      Only relevant for PubMed references.
     multivalued: true
     domain: Reference
     range: string
 
   pubmed_publication_status:
-    description: status of the publication
+    description: >-
+      Status of the publication at PubMed.
+      Only relevant for PubMed references.
     multivalued: false
     domain: Reference
     range: pubmed_publication_status_enum
 
   abstract:
     description: >-
       The author summary of the publication.
-    multivalued: false
-    domain: Reference
-    range: string
-
-  citation:
-    description: >-
-      A short form way of referring to a publication, typically consisting of
-      the first author's name, publication date, and journal (title, vol, pages).
+      From PubMed otherwise from Mod or manual reference creation.
     multivalued: false
     domain: Reference
     range: string
 
   issue_name:
+    description: >-
+      The issue name.
+      From PubMed otherwise from Mod or manual reference creation.
     multivalued: false
     domain: Reference
     range: string
 
   category:
+    description: >-
+      The alliance category type.  Only relevant at Alliance.
     multivalued: false
     domain: Reference
     range: reference_category_enum
 
   keywords:
+    description: >-
+      Keywords tagging a publication.  Aggregation of PubMed and ZFIN,
+      editable at Alliance.
     multivalued: true
-    description: keywords tagging a publication
     range: string
   
   language:
+    description: Language of the reference.  Aggregation of PubMed and FlyBase,
+      editable at Alliance.
     multivalued: false
-    description: Language of the reference
     domain: Reference
     range: string
 
   merged_into_id:
-    multivalued: false
     description: ID that used to refer to this reference
+    multivalued: false
     domain: Reference
     range: uriorcurie
 
-  summary:
-    description: >-
-    domain: Reference
-    range: string
-
-  copyright_date:
-    description: >-
-    domain: Reference
-    range: date
-
   authors:
+    description: >-
+      Ordered author entities for this publication.  An Author is associated
+      with only one publication.  A Person can be associated with multiple 
+      publications.
     multivalued: true
     domain: Reference
     range: AuthorReference
 
   corresponding_author:
-    description: >-
-    range: Reference
+    description: Indicates if the author is a corresponding author.
+    range: boolean
     domain: Reference
 
-
-  initials:
-    description: >-
+  first_author:
+    description: Indicates if the author is a first author.
+    range: boolean
     domain: Reference
```

Comment:
```suggestion
    domain: AuthorReference
```

Reply:
Yes, I agree that those two slots (`first_author`, `corresponding_author`) should be `multivalued: false`. Sorry for the confusion. I had made comments to the contrary earlier, but deleted them almost immediately once I realized that those two slots have `domain: AuthorReference` rather than `domain: Reference`.